Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
-RUN adduser -S dr4ftuser
-RUN chown dr4ftuser -R .
-USER dr4ftuser
+COPY --chown=node package.json .
+COPY --chown=node package-lock.json .
 
 # Install the dependencies
-RUN npm ci
+RUN npm ci --ignore-scripts
+
+# Update card database
+COPY --chown=node backend/core backend/core
+COPY --chown=node config/ config/
+COPY --chown=node scripts/ scripts/
+RUN npm run download_allsets \
+ && npm run download_booster_rules \
+ && chown node -R data/
+
+# Add sources to /app
```

Comment:
I tried to order these in such a way that more frequently-changing files would be copied in later layers so the layer cache invalidation would only apply to lower layers. Open to suggestions.

Also, I think the `app.json` file could be excluded here.